applications/sudokumb/README.md

```diff
@@ -0,0 +1,142 @@
+3.  [Click here](https://console.cloud.google.com/flows/enableapi?apiid=datastore.googleapis.com&showconfirmation=true)
+    to enable the Google Cloud Datastore API for your project.
+
+4.  [Click here](https://console.cloud.google.com/flows/enableapi?apiid=pubsub.googleapis.com&showconfirmation=true)
+    to enable the Google Stackdriver Cloud Pub/Sub API for your project.
+
+5.  [Click here](https://console.cloud.google.com/flows/enableapi?apiid=cloudkms.googleapis.com&showconfirmation=true) 
+    to enable Google Cloud Key Management Service for your project.
+
+
+6.  Edit [`WebApp/appsettings.json`](WebApp/appsettings.json).
+
+    Replace `YOUR-PROJECT-ID` with your Google project id.
+
+7.  Edit [`WebSolver/appsettings.json`](WebSolver/appsettings.json).
+
+    Replace `YOUR-PROJECT-ID` with your Google project id.
+
+## ![PowerShell](../../appengine/flexible/.resources/powershell.png) Using PowerShell
+
+### Run Locally
+
+1.	To run locally, run the WebApp and the WebSolver:
+    ```powershell
+    PS > dotnet run -p WebApp/WebApp.csproj
+    Hosting environment: Production
+    Content root path: /usr/local/google/home/<redacted>/gitrepos/dotnet-docs-samples/applications/sudokumb/WebApp
+    Now listening on: http://localhost:5000
+    Application started. Press Ctrl+C to shut down.
+    ```
+
+2.	In a separate window:
+    ```powershell
+    PS > $env:ASPNETCORE_URLS = 'http://localhost:5001'
+    PS > dotnet run -p WebSolver/WebSolver.csproj
+    Hosting environment: Production
+    Content root path: /usr/local/google/home/<redacted>/gitrepos/dotnet-docs-samples/applications/sudokumb/WebSolver
+    Now listening on: http://localhost:5001
+    Application started. Press Ctrl+C to shut down.
+    ```
+### Deploy to App Engine
+
+1.  Install the [Google Cloud SDK](http://cloud.google.com/sdk).
+
+2.  [Click here](https://console.cloud.google.com/appengine) to create an
+    App Engine Application.
+
+3.  Run the script to add Kms Permissions to the App Engine service account:
+    ```powershell
+    PS> .\Add-KmsPermissionsToAppEngine
+    ```
+    
+2.	Publish the projects:
+    ```powershell
+    PS > dotnet publish -c Release Sudokumb.sln
+    ```
+
+2.  Deploy WebApp with `gcloud`:
+    ```powershell
+    PS > gcloud beta app deploy WebApp/bin/Release/netcoreapp2.0/publish/app.yaml
+    ```
+
+3.  Deploy WebSolver with `gcloud`:
+    ```powershell
+    PS > gcloud beta app deploy WebSolver/bin/Release/netcoreapp2.0/publish/app.yaml
+    ```
+
+
+## ![Visual Studio](../../appengine/flexible/.resources/visual-studio.png) Using Visual Studio
+
+### Run Locally
+
+1.  Open ***Sudokumb.sln**.
+
+2.  In the **Solution Explorer** window, right-click the **Sudokumb** project 
+    and choose **Set StartUp Projects...".
+
+3.  Click **Multiple startup projects:**.  Change the **Action** from **None**
+    to **Start** for the projects **WebApp** and **WebSolver**.
+
+4.  Click **OK** to close the **Solution 'Sudokumb' Propert Pages** window.
```

Comment:
Update to be:

Click **OK** to close the **Solution 'Sudokumb' Property Pages** window.

Reply:
Done.